server/app.js

```diff
@@ -200,29 +201,31 @@ module.exports = function createApp({
   // Don't cache dynamic resources
   app.use(noCache())
 
-  // Request logging
-  app.use(
-    expressWinston.logger({
-      winstonInstance: logger,
-      meta: true,
-      dynamicMeta(req, res) {
-        const meta = {
-          userEmail: req.user ? req.user.username : null,
-          requestId: req.id,
-          sessionTag: req.user ? req.user.sessionTag : null,
```

Comment:
don't think this could have been working?!

Reply:
I've removed it completely and replaced with middleware that logs path, username and role as custom dimensions.